public/dorequest.php

```diff
@@ -237,15 +237,6 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
 
             PlayerSessionHeartbeat::dispatch($user, Game::find($gameID), $activityMessage);
 
-            // legacy rich presence support (deprecated - see ResumePlayerSession)
-            if (isset($activityMessage) && $user->LastGameID == $gameID) {
-                $user->RichPresenceMsg = utf8_sanitize($activityMessage);
-                $user->RichPresenceMsgDate = Carbon::now();
-            }
-
-            $user->LastLogin = Carbon::now();
-            $user->save();
```

Comment:
last login update should be handled by PlayerSessionHeartbeat.
verify that this is actually the case.